packages/frontend/src/i18n/pl.yaml

```diff
@@ -3,32 +3,33 @@ election_home:
   # start_time example: https://bettervoting.com/tcpwth
   start_time: '{{capital_election}} Wybory zaczynają się {{datetime, datetime}}'
   end_time: '{{capital_election}} Wybory kończą się {{datetime, datetime}}'
-  ended_time: '{{capital_election}} ended on {{datetime, datetime}}'
-  vote: Vote
-  or_view_results: or view results
-  ballot_submitted: '{{capital_ballot}} Submitted'
-  view_results: View Results
-  drafted: This election is still being drafted
-  archived: This election has been archived
+  ended_time: '{{capital_election}} zakończyły się {{datetime, datetime}}'
+  vote: Głosuj
+  or_view_results: lub wyświetl wyniki
+  ballot_submitted: '{{capital_ballot}} Karta głosowania wprowadzona'
+  view_results: Wyświetl wyniki 
+  drafted: Te wybory są jeszcze w fazie projektowania
+  archived: Te wybory zostały zarchiwizowane
 
 # Ballot (ex. https://bettervoting.com/j87vqp/vote)
 ballot:
   this_election_uses: This {{election}} will use {{voting_method}} to elect {{spelled_count}} $t(winner.winner).
-  instructions_checkbox: I have read the instructions
-  learn_more: Learn more about {{voting_method}}
-  previous: Previous
-  next: Next
-  submit_ballot: Submit
-  submitting: Submitting...
+  Te {{election}} wykorzystują metodę głosowania {{voting_method}}, aby wybrać {{spelled_count}} $t(winner.winner)
```

Comment:
I think this link needs to be applied above after the "this_election_uses"

Reply:
You should be able to fix this by going into the "Files Changed" tab and fixing the entry

![image](https://github.com/user-attachments/assets/bc5b4760-8f73-4e81-8359-160e39603518)